insights-ansible-playbook-verifier.spec

```diff
@@ -26,17 +26,19 @@ playbooks generated by Red Hat Insights.
 %pyproject_wheel
 
 %install
-%pyproject_install insights_ansible_playbook_lib insights_ansible_playbook_verifier
+%pyproject_install insights_ansible_playbook_lib insights_ansible_playbook_verifier insights_ansible_playbook_signer
 mkdir -p %{buildroot}/%{_libexecdir}
 mv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_libexecdir}
+mv %{buildroot}%{_bindir}/insights-ansible-playbook-signer %{buildroot}%{_libexecdir}
 mkdir -p %{buildroot}/%{_localstatedir}/lib/%{name}/
-%pyproject_save_files insights_ansible_playbook_lib insights_ansible_playbook_verifier
+%pyproject_save_files insights_ansible_playbook_lib insights_ansible_playbook_verifier insights_ansible_playbook_signer
 
 %check
-%pyproject_check_import insights_ansible_playbook_lib insights_ansible_playbook_verifier
+%pyproject_check_import insights_ansible_playbook_lib insights_ansible_playbook_verifier insights_ansible_playbook_signer
 
 %files -n %{name} -f %{pyproject_files}
 %{_libexecdir}/%{name}
+%{_libexecdir}/insights-ansible-playbook-signer
```

Comment:
hm I don't think the signer ought to be shipped with the verifier; it is mostly a developer tool

Reply:
This was meant to be temporary. Once this patch was in, before we started packaging it into downstream distributions, we'd split this off into a separate package. I didn't want to do this in one giant step, the spec file may need further cleanups to be easier to maintain long-term.

Reply:
Then I'd not build the signer for now during RPM builds, and we can add it later as you correctly proposed.